### DISCUSSION TITLE
the active slip system in HCP alloys

### DISCUSSION THREAD

**Discussion by zhangz95** [mtex-toolbox/mtex#574] [Ask Anything]
In EBSD-assisted slip trace analysis of Mg alloys, we need a code to express  the 12 possible traces and Schmid factors for each orientation in which a trace was detected and determination of the active slip system. Please help me, thank you!

**Comment by kilir** [comment]
Hi,
have you had a look here: https://mtex-toolbox.github.io/SchmidtFactor.html ?
Cheers,
Rüdiger

**Reply by zhangz95** [reply]
Dear Rüdiger，
Thank you very much for your reply. I can calculate the possible slip system according to your answer. However，I want to go further and get this kind of projection like graph, like the last one in the link：https://ars.els-cdn.com/content/image/1-s2.0-S1359645414007496-gr1.jpg
Cheers.
Zhang

**Reply by 2mapril** [reply]
so do you have the mtex code for 5 slip system for hcp,i will appreciate you very much if you can give me an answer!!! (mail:1022107270@qq.com)

**Comment by ralfHielscher** [comment]
For a slip system `sS` you get the trace by `sS.trace`

Ralf.
